packages/models-library/src/models_library/projects_nodes.py

```diff
@@ -87,7 +88,7 @@ class Node(BaseModel):
     run_hash: Optional[str] = Field(
         None,
         description="the hex digest of the resolved inputs +outputs hash at the time when the last outputs were generated",
-        examples=["a4337bc45a8fc544c03f52dc550cd6e1e87021bc896588bd79e901e2"],
+        example=["a4337bc45a8fc544c03f52dc550cd6e1e87021bc896588bd79e901e2"],
```

Comment:
I start using more complete examples instead of field examples. 
Checkout the [model_cls_examples fixture](https://github.com/ITISFoundation/osparc-simcore/blob/a53b5158b16d357db8fe43aac3092f23a473f69b/services/api-server/tests/unit/conftest.py#L260) and how is it used in the ``test_model_schemas_*.py`` test suites

In any case, we could also extend the fixture to "capture" and compose the field examples as well.